bin/inx-chronicle/src/config.rs

```diff
@@ -9,8 +9,17 @@ use thiserror::Error;
 
 #[derive(Error, Debug)]
 pub enum ConfigError {
+    #[cfg(feature = "api")]
+    #[error(transparent)]
+    Api(#[from] crate::api::ConfigError),
     #[error("failed to read file: {0}")]
     FileRead(std::io::Error),
+    #[cfg(feature = "api")]
+    #[error("failed to decode key: {0}")]
+    KeyDecode(libp2p_core::identity::error::DecodingError),
+    #[cfg(feature = "api")]
+    #[error("failed to read key bytes")]
+    KeyRead,
```

Comment:
Do those need to be separate from the `Api` error? (They have the same feature gate, so I wonder if those variants would either be better placed as variants of `crate::api::ConfigError`, or if we should remove the api feature gate?